pkg/kubelet/volumemanager/volume_manager_test.go

```diff
@@ -124,9 +125,16 @@ func TestInitialPendingVolumesForPodAndGetVolumesInUse(t *testing.T) {
 	// delayed claim binding
 	go delayClaimBecomesBound(kubeClient, claim.GetNamespace(), claim.ObjectMeta.Name)
 
-	err = manager.WaitForAttachAndMount(pod)
+	err = wait.Poll(100*time.Millisecond, 1*time.Second, func() (bool, error) {
```

Comment:
without this wait.Poll, the test should be flaky?

Reply:
Yes, because the PVC is bound after a short delay (see above) and the first few calls return error that the PVC is not bound yet.